Jellyfin.Server.Implementations/DisplayPreferencesManager.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Linq;
+using Jellyfin.Data.Entities;
+using MediaBrowser.Controller;
+
+namespace Jellyfin.Server.Implementations
+{
+    /// <summary>
+    /// Manages the storage and retrieval of display preferences through Entity Framework.
+    /// </summary>
+    public class DisplayPreferencesManager : IDisplayPreferencesManager
+    {
+        private readonly JellyfinDbProvider _dbProvider;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DisplayPreferencesManager"/> class.
+        /// </summary>
+        /// <param name="dbProvider">The Jellyfin db provider.</param>
+        public DisplayPreferencesManager(JellyfinDbProvider dbProvider)
+        {
+            _dbProvider = dbProvider;
+        }
+
+        /// <inheritdoc />
+        public DisplayPreferences GetDisplayPreferences(Guid userId, string client)
+        {
+            var dbContext = _dbProvider.CreateContext();
```

Comment:
Should this be wrapped in `using`?

Reply:
Good catch, I wrote this before the memory leak was fixed